src/csvcubed/cli/inspect/metadataprinter.py

```diff
@@ -203,13 +203,16 @@ def generate_codelist_results(self):
 
         Member of :class:`./MetadataPrinter`.
         """
-        csvw_state = self.state.csvw_state
+        csvw_inspector = self.state.csvw_inspector
         self.result_code_list_cols = select_codelist_cols_by_csv_url(
-            csvw_state.rdf_graph, self.primary_csv_url
+            csvw_inspector.rdf_graph, self.primary_csv_url
         )
         # Retrieving the primary key column names of the code list to identify the unique identifier
-        result_primary_key_col_names_by_csv_url: PrimaryKeyColNamesByDatasetUrlResult = select_primary_key_col_names_by_csv_url(
-            csvw_state.rdf_graph, self.primary_csv_url
+        # result_primary_key_col_names_by_csv_url: PrimaryKeyColNamesByDatasetUrlResult = select_primary_key_col_names_by_csv_url(
+        #     csvw_inspector.rdf_graph, self.primary_csv_url
+        # )
+        result_primary_key_col_names_by_csv_url: PrimaryKeyColNamesByDatasetUrlResult = (
+            self.state.csvw_inspector.get_codelist_primary_key_by_csv_url()
```

Comment:
Can this information (the primary key columns) be tacked on to the `get_table_schema_properties` query?

At any rate, I think the name of it needs to have the worse `codelist` removed since it isn't specifically about code lists (if it were it'd be on the CodeListInspector).

Reply:
Done